flink-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/streaming/connectors/cassandra/CassandraAppendTableSink.java

```diff
@@ -80,9 +81,15 @@ public void emitDataStream(DataStream<Row> dataStream) {
 			CassandraSink.addSink(dataStream)
 				.setClusterBuilder(this.builder)
 				.setQuery(this.cql)
-				.build();
+				.build()
+				.name(explainSink());
```

Comment:
Call `TableConnectorUtil.genRuntimeName(this.getClass().getSimpleName(), fieldNames)` directly?